packages/core/r4b/bundle-navigator.ts

```diff
@@ -218,18 +224,29 @@ export class BundleNavigator<
     >[];
   }
 
+  /**
+   * Get all the resources of the bundle.
+   */
+  public bundleResources() {
```

Comment:
I think making this a [getter](https://www.typescriptlang.org/docs/handbook/2/classes.html#getters--setters) might make the API more fluid:

```suggestion
  public get resources() {
```

Example usage:

```typescript
navigator.resources.map(res => ...);
```